bindings/kvp/src/main/java/org/n52/sos/decode/kvp/AbstractKvpDecoder.java

```diff
@@ -337,13 +339,13 @@ protected Time parseTime(String parameterValue, String parameterName) throws Cod
         return namespaces;
     }
 
+
     private TemporalFilter createTemporalFilterFromValue(String value, String valueReference)
             throws OwsExceptionReport, DateTimeParseException {
         TemporalFilter temporalFilter = new TemporalFilter();
         temporalFilter.setValueReference(valueReference);
         String[] times = value.split("/");
-
-        if (times.length == 1) {
+        if (times.length == 1 && !temporalFilter.getOperator().equals(TimeOperator.TM_During)) {
```

Comment:
@BrianGouge-AI : To your other question: the filters "first" and "latest" are parsed in the times.length==1 part of the if condition. So my changes do not affect that. Will test it anyways.